src/lib/select/select.ts

```diff
@@ -22,7 +22,35 @@ import {Subscription} from 'rxjs/Subscription';
 import {transformPlaceholder, transformPanel, fadeInContent} from './select-animations';
 import {ControlValueAccessor, NgControl} from '@angular/forms';
 import {coerceBooleanProperty} from '../core/coersion/boolean-property';
-import {ConnectedOverlayPositionChange} from '../core/overlay/position/connected-position';
+import {ConnectedOverlayDirective} from '../core/overlay/overlay-directives';
+import {ViewportRuler} from '../core/overlay/position/viewport-ruler';
+
```

Comment:
Maybe add a high level comment here explaining that you need to know a handful of style values here in order to properly align the selected item on open.